# FrontierArray.msg - Custom message for Frontier Array structure

Frontier[] frontiers